### PR TITLE
About page with router

### PR DESCRIPTION
**Pull request by alvikalvik**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "axios": "0.21.4",
     "core-js": "3.2.1",
     "history": "4.10.1",
+    "lodash": "^4.17.21",
```
Comment (ximet) [inline package.json]: for what lodash? I didn't see anything import from lodash

Comment (alvikalvik) [reply package.json]: I use debounce from lodash in previous PR. In this PR I just reinstalled lodash (it seems it was installed improperly first time, I received warning during one of builds).

Comment (anton-vlasik) [reply package.json]: you can install individual packages: in this case lodash.debounce

Comment (alvikalvik) [reply package.json]: Great, thanks. Replaced lodash  with lodash.debounce

**Review by ximet — APPROVED** [review]

**Pull request merged**